docs/install.md

```diff
@@ -7,7 +7,7 @@ You can install Presidio as a service in [Kubernetes](https://kubernetes.io/) or
 ```sh
 # Build the images
 
-$ export DOCKER_REGISTRY=presidio
+$ export DOCKER_REGISTRY=mcr.microsoft.com
```

Comment:
I think this is wrong. as 'presidio' is not really an ACR.
meaning this export line is overriding the default ACR so that all images will be pulled locally.
so this change is wrong in this context.

Reply:
@eladiw Done